src/VerifyExamples/Comment.elm

```diff
@@ -0,0 +1,41 @@
+module VerifyExamples.Comment exposing (Comment, parse)
+
+import Regex exposing (HowMany(..), Regex)
+
+
+type alias Comment =
+    { comment : String
+    , functionName : String
+    }
+
+
+parse : String -> List Comment
+parse =
+    Regex.find All commentRegex
+        >> List.filterMap (toComment << .submatches)
+
+
+toComment : List (Maybe String) -> Maybe Comment
+toComment matches =
+    case matches of
+        (Just comment) :: (Just functionName) :: _ ->
+            Just { comment = comment, functionName = functionName }
+
+        _ ->
+            Nothing
+
+
+commentRegex : Regex
+commentRegex =
+    Regex.regex <|
+        String.join ""
+            [ "({-[^]*?-})" -- anything between comments
```

Comment:
@eeue56 does this help?

Reply:
`String.concat` is a useful little `String.join ""` alternative - less thinking about what's going on here.

Reply:
d'oh! will fix.